### PR TITLE
Feature/rdf/examples to v3 rc02

### PR DESCRIPTION
**Pull request by changqin26**
There are two open points when updating the examples.
1. "Class Identifier" express just a string now. But in the Spec, the range of Identification/id is still a class-Identifier, not "String". And in the example, I can only write a string as the value. In conclusion, the example cannot pass the Shacl validation.
2. I don't really get the ideas of SubmodelElementList on the page 82 from line 7 to 12 in the Spec, so I am not sure if I updated the content in Complete_example.ttl from line 216 to 401 correctly.

### CODE REVIEW

File: schemas/rdf/examples/Asset_Example.ttl [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 
 <http://customer.com/assets/KHBVZJSQKIY>
-    rdf:type <https://admin-shell.io/aas/3/0/RC01/Asset> ;
-    <https://admin-shell.io/aas/3/0/RC01/Referable/idShort> "ServoDCMotor"^^xsd:string ;
+    rdf:type <https://admin-shell.io/aas/3/0/RC02/Asset> ;
```
Comment (BirgitBoss) [inline schemas/rdf/examples/Asset_Example.ttl:9]: No Asset in V3.0RC02 any longer

File: schemas/rdf/examples/AAS_Reference_shortExample.ttl [hunk 2]
```diff
@@ -22,34 +22,25 @@
...
-              a <https://admin-shell.io/aas/3/0/RC01/Key> ;
-              <https://admin-shell.io/aas/3/0/RC01/Key/type> <https://admin-shell.io/aas/3/0/RC01/IdentifiableElements/Asset> ;
-              <https://admin-shell.io/aas/3/0/RC01/Key/value> "http://customer.com/assets/KHBVZJSQKIY"^^xsd:string ;
-              <https://admin-shell.io/aas/3/0/RC01/Key/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Iri>
+<http://customer.com/aas/9175_7013_7091_9168> rdf:type <https://admin-shell.io/aas/3/0/RC02/AssetAdministrationShell> ;
+    <https://admin-shell.io/aas/3/0/RC02/Identifiable/id> "http://customer.com/aas/9175_7013_7091_9168"^^xsd:string ;
+    <https://admin-shell.io/aas/3/0/RC02/Referable/idShort> "a9175_7013_7091_9168"^^xsd:string ;
+    <https://admin-shell.io/aas/3/0/RC02/AssetAdministrationShell/assetInformation> [
+        a <https://admin-shell.io/aas/3/0/RC02/AssetInformation> ;
+        <https://admin-shell.io/aas/3/0/RC02/AssetInformation/globalAssetId> [
```
Comment (BirgitBoss) [inline schemas/rdf/examples/AAS_Reference_shortExample.ttl:30]: globalAssetId is a global reference, not a model reference

File: schemas/rdf/examples/AAS_Reference_shortExample.ttl [hunk 3]
```diff
@@ -22,34 +22,25 @@
...
 
 <http://customer.com/assets/KHBVZJSQKIY>
-    rdf:type <https://admin-shell.io/aas/3/0/RC01/Asset> ;
-    <https://admin-shell.io/aas/3/0/RC01/Identifiable/identification> [
-       a <https://admin-shell.io/aas/3/0/RC01/Identifier> ;
-       <https://admin-shell.io/aas/3/0/RC01/Identifier/identifier> "http://customer.com/assets/KHBVZJSQKIY"^^xsd:string ;
-       <https://admin-shell.io/aas/3/0/RC01/Identifier/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Iri> ;
-    ] ;
-    <https://admin-shell.io/aas/3/0/RC01/Referable/idShort> "KHBVZJSQKIY"^^xsd:string ;
+    rdf:type <https://admin-shell.io/aas/3/0/RC02/Asset> ;
```
Comment (BirgitBoss) [inline schemas/rdf/examples/AAS_Reference_shortExample.ttl:43]: no Asset any longer

File: schemas/rdf/examples/AssetAdministrationShell_Example.ttl [hunk 4]
```diff
@@ -9,65 +9,56 @@
...
-              <https://admin-shell.io/aas/3/0/RC01/Reference/key> [
-                a <https://admin-shell.io/aas/3/0/RC01/Key> ;
-                <https://admin-shell.io/aas/3/0/RC01/Key/type> <https://admin-shell.io/aas/3/0/RC01/IdentifiableElements/Asset> ;
-                <https://admin-shell.io/aas/3/0/RC01/Key/value> "http://customer.com/assets/KHBVZJSQKIY"^^xsd:string ;
-                <https://admin-shell.io/aas/3/0/RC01/Key/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Iri> ;
+      <https://admin-shell.io/aas/3/0/RC02/AssetAdministrationShell/assetInformation> [
+          a <https://admin-shell.io/aas/3/0/RC02/AssetInformation> ;
+          <https://admin-shell.io/aas/3/0/RC02/AssetInformation/assetKind> <https://admin-shell.io/aas/3/0/RC02/AssetKind/Instance> ;
+          <https://admin-shell.io/aas/3/0/RC02/AssetInformation/globalAssetId> [
+            a  <https://admin-shell.io/aas/3/0/RC02/ModelReference> ;
```
Comment (BirgitBoss) [inline schemas/rdf/examples/AssetAdministrationShell_Example.ttl:21]: see above: it is a global reference

File: schemas/rdf/examples/AssetAdministrationShell_Example.ttl [hunk 5]
```diff
@@ -9,65 +9,56 @@
...
-            a <https://admin-shell.io/aas/3/0/RC01/Key> ;
-            <https://admin-shell.io/aas/3/0/RC01/Key/type> <https://admin-shell.io/aas/3/0/RC01/IdentifiableElements/Submodel>  ;
-            <https://admin-shell.io/aas/3/0/RC01/Key/value> "http://i40.customer.com/type/F13E8576F6488342"^^xsd:string ;
-            <https://admin-shell.io/aas/3/0/RC01/Key/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Iri>;
+      <https://admin-shell.io/aas/3/0/RC02/AssetAdministrationShell/submodel> [
+          a <https://admin-shell.io/aas/3/0/RC02/ModelReference> ;
+          <https://admin-shell.io/aas/3/0/RC02/ModelReference/key> [
+            a <https://admin-shell.io/aas/3/0/RC02/Key> ;
+            <https://admin-shell.io/aas/3/0/RC02/Key/type> <https://admin-shell.io/aas/3/0/RC02/KeyElements/Submodel>  ;
+            <https://admin-shell.io/aas/3/0/RC02/Key/value> "http://i40.customer.com/type/F13E8576F6488342"^^xsd:string ;
```
Comment (BirgitBoss) [inline schemas/rdf/examples/AssetAdministrationShell_Example.ttl:35]: we decided to only recommend https IRIs

File: schemas/rdf/examples/AssetAdministrationShell_Example.ttl [hunk 6]
```diff
@@ -9,65 +9,56 @@
...
-      <https://admin-shell.io/aas/3/0/RC01/AssetAdministrationShell/submodel> [
-          a <https://admin-shell.io/aas/3/0/RC01/Reference> ;
-          <https://admin-shell.io/aas/3/0/RC01/Reference/key> [
-            a <https://admin-shell.io/aas/3/0/RC01/Key> ;
-            <https://admin-shell.io/aas/3/0/RC01/Key/type> <https://admin-shell.io/aas/3/0/RC01/IdentifiableElements/Submodel>  ;
-            <https://admin-shell.io/aas/3/0/RC01/Key/value> "http://i40.customer.com/type/F13E8576F6488342"^^xsd:string ;
-            <https://admin-shell.io/aas/3/0/RC01/Key/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Iri>;
+      <https://admin-shell.io/aas/3/0/RC02/AssetAdministrationShell/submodel> [
+          a <https://admin-shell.io/aas/3/0/RC02/ModelReference> ;
+          <https://admin-shell.io/aas/3/0/RC02/ModelReference/key> [
```
Comment (BirgitBoss) [inline schemas/rdf/examples/AssetAdministrationShell_Example.ttl:32]: really key or keys?

File: schemas/rdf/examples/AssetAdministrationShell_Example.ttl [hunk 7]
```diff
@@ -9,65 +9,56 @@
...
           ]
       ];
-      <https://admin-shell.io/aas/3/0/RC01/Identifiable/identification> [
-         a <https://admin-shell.io/aas/3/0/RC01/Identifier> ;
-         <https://admin-shell.io/aas/3/0/RC01/Identifier/identifier> "http://customer.com/aas/9175_7013_7091_9168"^^xsd:string ;
-         <https://admin-shell.io/aas/3/0/RC01/Identifier/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Iri> ;
-      ] ;
-      <https://admin-shell.io/aas/3/0/RC01/Referable/category> "CONSTANT"^^xsd:string ;
+      <https://admin-shell.io/aas/3/0/RC02/Identifiable/id> "http://customer.com/aas/9175_7013_7091_9168"^^xsd:string ;
+      <https://admin-shell.io/aas/3/0/RC02/Referable/category> "CONSTANT"^^xsd:string ;
```
Comment (BirgitBoss) [inline schemas/rdf/examples/AssetAdministrationShell_Example.ttl:63]: no category for submodel

File: schemas/rdf/examples/KapitalVerwaltungsschaleExample.ttl [hunk 8]
```diff
@@ -8,24 +8,23 @@
...
-         <https://admin-shell.io/aas/3/0/RC01/Key/value> "0173-1#02-BAA120#008"^^xsd:string ;
-         <https://admin-shell.io/aas/3/0/RC01/Key/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Irdi> ;
+    a <https://admin-shell.io/aas/3/0/RC02/Property> ;
+    <https://admin-shell.io/aas/3/0/RC02/Property/valueType> "xsd:integer" ;
+    <https://admin-shell.io/aas/3/0/RC02/Property/value> "5000"^^xsd:integer ;
+    <https://admin-shell.io/aas/3/0/RC02/Referable/idShort> "MaxRotationSpeed"^^xsd:string ;
+    <https://admin-shell.io/aas/3/0/RC02/Referable/category> "PARAMETER"^^xsd:string ;
+    <https://admin-shell.io/aas/3/0/RC02/HasKind/kind> <https://admin-shell.io/aas/3/0/RC02/ModelingKind/Instance>  ;
+    <https://admin-shell.io/aas/3/0/RC02/HasSemantics/semanticId> [
+       a <https://admin-shell.io/aas/3/0/RC02/ModelReference> ;
```
Comment (BirgitBoss) [inline schemas/rdf/examples/KapitalVerwaltungsschaleExample.ttl:22]: shall be GlobalReference

File: schemas/rdf/examples/KapitalVerwaltungsschaleExample.ttl [hunk 9]
```diff
@@ -8,24 +8,23 @@
...
+       <https://admin-shell.io/aas/3/0/RC02/Referable/idShort> "AReference"^^xsd:string ;
```
Comment (BirgitBoss) [inline schemas/rdf/examples/KapitalVerwaltungsschaleExample.ttl:23]: Why does a semanticId have an idShort (it is not a Referable)?

File: schemas/rdf/examples/Submodel_SubmodelElement_Example.ttl [hunk 10]
```diff
@@ -1,71 +1,40 @@
...
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 
 
 <http://i40.customer.com/type/F13E8576F6488342> rdf:type aas:Submodel ;
      rdfs:label "Identification"^^xsd:string ;
-     <https://admin-shell.io/aas/3/0/RC01/Referable/idShort> "Identification"^^xsd:string ;
-     <https://admin-shell.io/aas/3/0/RC01/Referable/category> "CONSTANT"^^xsd:string ;
+     <https://admin-shell.io/aas/3/0/RC02/Referable/idShort> "Identification"^^xsd:string ;
+     <https://admin-shell.io/aas/3/0/RC02/Referable/category> "CONSTANT"^^xsd:string ;
```
Comment (BirgitBoss) [inline schemas/rdf/examples/Submodel_SubmodelElement_Example.ttl:12]: no category for submodel

File: schemas/rdf/examples/Submodel_SubmodelElement_shortExample.ttl [hunk 11]
```diff
@@ -1,66 +1,35 @@
...
-                  <https://admin-shell.io/aas/3/0/RC01/Key/idType> <https://admin-shell.io/aas/3/0/RC01/IdentifierType/Iri>
-              ] ;
+          <https://admin-shell.io/aas/3/0/RC02/Referable/idShort> "Manufacturer"^^xsd:string ;
+          <https://admin-shell.io/aas/3/0/RC02/Property/valueType>  "xsd:string"^^xsd:string ;
+          <https://admin-shell.io/aas/3/0/RC02/Property/valueId> [
+              a <https://admin-shell.io/aas/3/0/RC02/GlobalReference> ;
+              <https://admin-shell.io/aas/3/0/RC02/GlobalReference/value> "http://i40.customer.com/type/F13E8576F6488342/Manufacturer"^^xsd:string ;
           ] ;
-          <https://admin-shell.io/aas/3/0/RC01/Property/value> "http://i40.customer.com/type/F13E8576F6488342/Manufacturer"^^xsd:string ;
+          <https://admin-shell.io/aas/3/0/RC02/Property/value> "http://i40.customer.com/type/F13E8576F6488342/Manufacturer"^^xsd:string ;
```
Comment (BirgitBoss) [inline schemas/rdf/examples/Submodel_SubmodelElement_shortExample.ttl:26]: The valueId is http://i40.customer.com/type/F13E8576F6488342/Manufacturer. So the value is <http://i40.customer.com/type/F13E8576F6488342/Manufacturer> or "X GmbH", isn'it it? It is very unprobable that the value is identical to the valueId

**Review by BirgitBoss — CHANGES_REQUESTED** [review]
see comments

no Asset any longer
value and valueId
no category for submodel etc., only for property 
Submodel Documentation consists of a SubmodelElementList of SubmodelElementStructs. Every single Document is a SubmodelElementStruct. But the set of all Documents is a SubmodelElementList.

rename file KapitalVerwaltungsschaleExample.ttl (German und it is no chapter any longer)

I do not get your comment "But in the Spec, the range of Identification/id is still a class-Identifier, not "String". " In the spec it is a string (but defined by separate atomic type).

File: schemas/rdf/examples/Asset_Example.ttl (hunk 1 above)
Comment (changqin26) [reply schemas/rdf/examples/Asset_Example.ttl]: The examples are deleted.

File: schemas/rdf/examples/AAS_Reference_shortExample.ttl (hunk 2 above)
Comment (changqin26) [reply schemas/rdf/examples/AAS_Reference_shortExample.ttl:30]: Corrected.

File: schemas/rdf/examples/AAS_Reference_shortExample.ttl (hunk 3 above)
Comment (changqin26) [reply schemas/rdf/examples/AAS_Reference_shortExample.ttl]: Deleted.

File: schemas/rdf/examples/AssetAdministrationShell_Example.ttl (hunk 4 above)
Comment (changqin26) [reply schemas/rdf/examples/AssetAdministrationShell_Example.ttl:21]: Corrected.

File: schemas/rdf/examples/AssetAdministrationShell_Example.ttl (hunk 6 above)
Comment (changqin26) [reply schemas/rdf/examples/AssetAdministrationShell_Example.ttl:32]: On page 87 of Spec, it is key, not keys.

File: schemas/rdf/examples/KapitalVerwaltungsschaleExample.ttl (hunk 9 above)
Comment (changqin26) [reply schemas/rdf/examples/KapitalVerwaltungsschaleExample.ttl]: Deleted.

File: schemas/rdf/examples/AssetAdministrationShell_Example.ttl (hunk 5 above)
Comment (changqin26) [reply schemas/rdf/examples/AssetAdministrationShell_Example.ttl]: Modified all http IRIs to https IRIs

File: schemas/rdf/examples/AssetAdministrationShell_Example.ttl (hunk 7 above)
Comment (changqin26) [reply schemas/rdf/examples/AssetAdministrationShell_Example.ttl]: deleted

File: schemas/rdf/examples/Submodel_SubmodelElement_Example.ttl (hunk 10 above)
Comment (changqin26) [reply schemas/rdf/examples/Submodel_SubmodelElement_Example.ttl]: Deleted

File: schemas/rdf/examples/Submodel_SubmodelElement_shortExample.ttl (hunk 11 above)
Comment (changqin26) [reply schemas/rdf/examples/Submodel_SubmodelElement_shortExample.ttl]: Yes, you are right. I fixed the property/value.

**Review by BirgitBoss — CHANGES_REQUESTED** [review]
I wonder: where is the SubmodelElementStruct? I did not find it. Did you remove it?
For many properties or other submodel elements the value is identical to the semanticId. For example MaxRoationSpeed Value is e.g. 5000 , MaxTorque is e.g. 1500 GLN is e.g. 1234567000005 ==> please check all values, I do not have your aasx that you take as Basis for the example to look up all the correct or realistic values

Product Designation is MultiLanguageProperty: the example value is an IRDI and just a string. So this seems not correct: how does the rdf schema request to do it?

**Comment by BirgitBoss** [comment]
We cannot yet merge these changes for V3.0RC02 because we before want to make a valid release for V3.0RC01 with its examples. Starting with V3.0RC02 all mapping rules and examples will be maintained in github (this means in this repo) only.

File: schemas/rdf/examples/Complete_Example.ttl [hunk 12]
```diff
@@ -8,1116 +8,832 @@
...
-                  ]
+             <https://admin-shell.io/aas/3/0/RC02/Referable/idShort> "GLN"^^xsd:string ;
+              a <https://admin-shell.io/aas/3/0/RC02/Property> ;
+              <https://admin-shell.io/aas/3/0/RC02/Property/valueType>  "xsd:string" ;
+              <https://admin-shell.io/aas/3/0/RC02/Property/valueId> [
+                  a <https://admin-shell.io/aas/3/0/RC02/GlobalReference> ;
+                  <https://admin-shell.io/aas/3/0/RC02/GlobalReference/value> "0173-1#02-AAY812#001"^^xsd:string ;
               ];
-              <https://admin-shell.io/aas/3/0/RC01/Property/value> "0173-1#02-AAY812#001"^^xsd:string ;
+              <https://admin-shell.io/aas/3/0/RC02/Property/value> "0173-1#02-AAY812#001"^^xsd:string ;
```
[the remark this answers is not in the document]
Comment (changqin26) [reply schemas/rdf/examples/Complete_Example.ttl:99]: Yes, you are right. Corrected.

**Pull request closed without merging**